pkg/framework/report.go

```diff
@@ -98,20 +105,22 @@ func getMainFailReason(message string) *ClusterCapacityReviewScheduleFailReason
 }
 
 func getResourceRequest(pod *api.Pod) *Resources {
-	result := Resources{
-		CPU:       resource.NewMilliQuantity(0, resource.DecimalSI),
-		Memory:    resource.NewQuantity(0, resource.BinarySI),
-		NvidiaGPU: resource.NewMilliQuantity(0, resource.DecimalSI),
-	}
+	result := Resources{}
+	resources := make(map[api.ResourceName]*resource.Quantity)
+
+	resources[api.ResourceMemory] = resource.NewMilliQuantity(0, resource.DecimalSI)
+	resources[api.ResourceCPU] = resource.NewQuantity(0, resource.BinarySI)
```

Comment:
Seems like a typo, you switched Memory and CPU, should be milliquantity for cpu and just quantity for memory.